rskj-core/src/test/java/co/rsk/peg/BridgeUtilsTest.java

```diff
@@ -2914,4 +2919,206 @@ private Federation getGenesisFederationForTest(BridgeConstants bridgeConstants,
 
         return federation;
     }
+
+    @Test
+    public void getMinimumPegInTxValue() {
+        ActivationConfig.ForBlock activations = mock(ActivationConfig.ForBlock.class);
+        // Before RSKIP219 activation
+        when(activations.isActive(ConsensusRule.RSKIP176)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP219)).thenReturn(false);
+
+        BridgeConstants bridgeConstants = bridgeConstantsRegtest;
+        Coin minimumPeginTxValue = bridgeConstants.getLegacyMinimumPeginTxValueInSatoshis();
+        assertEquals(
+            minimumPeginTxValue,
+            BridgeUtils.getMinimumPegInTxValue(activations, bridgeConstants)
+        );
+
+        bridgeConstants = bridgeConstantsMainnet;
+        minimumPeginTxValue = bridgeConstants.getLegacyMinimumPeginTxValueInSatoshis();
+        assertEquals(
+            minimumPeginTxValue,
+            BridgeUtils.getMinimumPegInTxValue(activations, bridgeConstants)
+        );
+
+        // After RSKIP219 activation
+        when(activations.isActive(ConsensusRule.RSKIP176)).thenReturn(true);
+        when(activations.isActive(ConsensusRule.RSKIP219)).thenReturn(true);
+
+        bridgeConstants = bridgeConstantsRegtest;
+        minimumPeginTxValue = bridgeConstantsRegtest.getMinimumPeginTxValueInSatoshis();
+        assertEquals(
+            minimumPeginTxValue,
+            BridgeUtils.getMinimumPegInTxValue(activations, bridgeConstants)
+        );
+
+        bridgeConstants = bridgeConstantsMainnet;
+        minimumPeginTxValue = bridgeConstants.getMinimumPeginTxValueInSatoshis();
+        assertEquals(
+            minimumPeginTxValue,
+            BridgeUtils.getMinimumPegInTxValue(activations, bridgeConstants)
+        );
+    }
+
+    private void isAnyUTXOAmountBelowMinimum_by_network(
```

Comment:
Man, you are definitely right. The current tests should be changed to test any given address instead of the federation address.

So yes, I will consider each test case you mentioned above.

Reply:
Done